src/views/docs-view/docs-view.module.css

```diff
@@ -36,10 +36,5 @@
 		& :global(.landing-hero) {
 			margin-right: 180px;
 		}
-
-		/* Add more space in between the first section after title and the version selector  */
-		& h1:first-child {
-			margin-bottom: 16px;
-		}
```

Comment:
In recent design specs, there were specific notes to ensure spacing between the breadcrumb and `h1` is `32px`, so have removed this styling exception for when the version selector is present (which is on most docs pages).

cc @braydenlove (i figure this might be worth confirming specifically 🙇)

Reply:
Perfecto, awesome work.